### DISCUSSION TITLE
Error during upload / File upload failed after upgrade from 2.5.201 to 2.5.272

### DISCUSSION THREAD

**Discussion by JanZeman** [requarks/wiki#4943] [Error / Bug Report] [answered]
Hello,

I just upgraded my wiki to v.2.5.272 and any time I try to upload any image I get the in red: Error during upload / File upload failed.
Luckily I run 2 wiki instances and the one I did not upgrade yet uploads the very same image without any problems.

![image](https://user-images.githubusercontent.com/1670470/151410697-0371e750-46b6-4f21-9d7d-f570cac32307.png)

Is this a known issue?

Best regards
Jan

**Comment by NGPixel** [comment, marked as answer]
Make sure to clear your browser cache + any proxy cache between you and the server.

**Reply by JanZeman** [reply]
Hi @NGPixel, thank you for the answer. Actually I tried that before posting, tried incognito tab and also other browser - with the same failure. By now I rollbacked the server and it works again but I will give it another attempt. Luckily my hosting provider allows me to rollback easily. Isn't there any option to enable any kind of a verbose / debug mode where the error message would produce more details about the error?

**Reply by NGPixel** [reply]
Press F12 to open the dev tools in your browser and go to the network tab. Try to upload a file again and click on the failed request. What's the server response?

**Reply by JanZeman** [reply]
Hello again @NGPixel , I repeated the whole upgrade process from scratch over the rollbacked version and this time it all worked. I can upload images again. My apologies for a false alarm - apparently the problem was in my infrastructure not in node.js code. Thank you! Jan
